pkgs/by-name/ze/zenmap/package.nix

```
@@ -0,0 +1,76 @@
+{
+  gobject-introspection,
+  gtk3,
+  lib,
+  nmap,
+  python3Packages,
+  wrapGAppsHook3,
+  xterm,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zenmap";
+  version = nmap.version;
+  pyproject = true;
+
+  src = nmap.src;
+
+  sourceRoot = "${nmap.name}/zenmap";
```

```suggestion
  prePatch = ''
    cd zenmap
  '';
```
`nmap.name` is not always nmap-7.95 (e.g. for cross or other configurations)